verilog/analysis/verilog_linter.cc

```diff
@@ -228,36 +247,54 @@ ViolationFixer::AnswerChoice ViolationFixer::InteractiveAnswerChooser(
       "P - show fixes applied in this file so far\n"
       "? - print this help and prompt again\n";
 
+  // If we have alternatives, show this in the short-menu.
+  std::string alternative_list;
+  if (size_t fix_count = violation.autofixes.size(); fix_count > 1) {
+    for (size_t i = 0; i < fix_count; ++i)
+      alternative_list += std::to_string(i + 1) + ",";
+  }
+
   for (;;) {
     const char c = verible::ReadCharFromUser(
         std::cin, std::cerr, verible::IsInteractiveTerminalSession(),
-        "Autofix is available. Apply? [y,n,a,d,A,D,p,P,?] ");
+        "Autofix is available. Apply? [" + alternative_list +
+            "y,n,a,d,A,D,p,P,?] ");
```

Comment:
maybe insert `1-9` among the choices?

Reply:
The available choices are in there. So if if there are two choices, it will `[1,2,y,n...]`